ads_plugin.py

```diff
@@ -86,10 +86,8 @@ def __init__(self, ortho, *args, **kwargs):
         load_mask_button.Bind(wx.EVT_BUTTON, self.on_load_mask_button)
         load_mask_button.SetToolTip(
             wx.ToolTip(
-                "Loads an existing axon or myelin mask into FSLeyes. "
-                "Please make sure the axon mask contains 'axon' in its name and "
-                "the myelin mask contains 'myelin' in its name. "
-                "The image must only contain 2 values: 0 for background and 255 for the region of interest."
+                "Loads an existing axonmyelin mask into FSLeyes. "
+                "The selected image should contain both the axon and myelin masks."
```

Comment:
Are there specific values for background, myelin and axon? (ex: 0/128/255). If so, should we indicate them?

Reply:
done in 00e23d7